### PR TITLE
[service.games.discord.richpresence] 1.0.0

### PR DESCRIPTION
**Pull request by NikosSiak**
### Description
This addon will display what game are you playing and what you are doing in the game at this moment. This is based on the https://github.com/xbmc/xbmc/pull/21286 PR
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: service.games.discord.richpresence/addon.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        name="Discord Rich Presence for RetroPlayer"
+        version="1.0.0"
+        provider-name="NikosSiak">
+    <requires>
+        <import addon="xbmc.python" version="3.0.1"/>
+    </requires>
+    <extension point="xbmc.service" library="default.py" />
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <source>https://github.com/NikosSiak/service.games.discord.richpresence</source>
```
Comment (enen92) [inline service.games.discord.richpresence/addon.xml:12]: Missing description and summary (en_GB), missing license (spdx identifier). See example:  https://github.com/xbmc/xbmc/blob/master/addons/service.xbmc.versioncheck/addon.xml

File: service.games.discord.richpresence/default.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import xbmc
```
Comment (enen92) [inline service.games.discord.richpresence/default.py:1]: usually it's a good practice to keep this file (as the main entrypoint) as small as possible. Reason is the fact this is the only file that is not compiled to bytecode.
Can you move this to `resources/lib/richpresence.py` and in this file simply import and run?

```
from resources.lib.richpresence import mystuff

mystuff.run()
```

There's a warning for this in addon checker

File: service.games.discord.richpresence/default.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+CLIENT_ID = '750797452453478629'
+
+if __name__ == "__main__":
+    monitor = xbmc.Monitor()
+    player = xbmc.Player()
+    discord = None
+    startTime = -1
+    gameStarted = False
+
+    while True:
```
Comment (enen92) [inline service.games.discord.richpresence/default.py:14]:
```
while not monitor.abortRequested():
    if monitor.abortRequested():
        break
    your stuff
    monitor.waitForAbort(5)
```

File: service.games.discord.richpresence/resources/lib/discordIPC.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
+import os
```
Comment (enen92) [inline service.games.discord.richpresence/resources/lib/discordIPC.py:1]: the dummy `__init__.py` is no longer needed in python3, can you drop it from the submission?

**Comment by basrieter** [comment]
Closed due to non-responsiveness.

**Pull request closed without merging**